src/components/OntologyBrowser/Test/SelectionTree.test.tsx

```diff
@@ -11,16 +11,31 @@ describe('In SelectionTree', () => {
   const checkedKeys: string[] = [];
   const targetKeys: string[] = [];
   const onItemSelect = jest.fn();
+  const onItemSelectAll = jest.fn();
   const dataSource: TreeNode[] = treeData
 
   beforeAll(() => {
+    Object.defineProperty(window, 'matchMedia', {
```

Comment:
okay, but I'll change that when my pr will be pushed...I extracted that into a utils